factorio.go

```diff
@@ -21,6 +26,12 @@ func factorioGetChecksum(url string) (string, error) {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode == http.StatusNotFound {
```

Comment:
From my experience, this download endpoint is quite unstable (or has improperly configured rate-limiting). We should include a wait-and-retry mechanism for the case that we need to retry the download.

Reply:
it retries automaticly, because of the instability issues you mentioned.